api/models/Community.js

```diff
@@ -146,6 +146,28 @@ module.exports = bookshelf.Model.extend(merge({
     })
     .fetchAll()
     .then(skills => skills.pluck('name'))
+  },
+
+  memberCount: function () {
+    return User.query(q => {
```

Comment:
it really rankles that there isn't a good way to do this based on the `hasMany` methods we've already defined.